cli/src/torii/routing.rs

```diff
@@ -204,8 +203,7 @@ enum Health {
     Healthy,
 }
 
-#[iroha_futures::telemetry_future]
```

Comment:
Just because it's not a particularly useful metric, I suppose?

Reply:
> Just because it's not a particularly useful metric,

Yes. But also because the healthcheck is supposed to be quick and lightweight. Because you're going through the telemetry subsystem with its own `Future` the way it used to be pessimised the performance quite a bit, and not just for the health check. Just because the health check is running in its own thread doesn't mean it can't clog the `async` pipeline.